luminoth/models/fasterrcnn/fasterrcnn_test.py

```diff
@@ -213,7 +215,7 @@ def testBasic(self):
         # Check that every rpn proposal has 4 coordinates + 1 batch index
         self.assertEqual(
             rpn_prediction['proposals'].shape[1],
-            5
+            4
         )
```

Comment:
We should change the comment to reflect the change.

Reply:
Fixed!